src/main/java/com/example/spacecatmarket/javawebkpi/domain/Cart.java

```diff
@@ -0,0 +1,14 @@
+package com.example.spacecatmarket.javawebkpi.domain;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
```

Comment:
Good but I would prefer to use @Value, for domain object. Our rule of thumb - if sth could be immutable, we will do it immutable